### ISSUE TITLE
Brew End Date/Tilt Data

### ISSUE THREAD

**Issue by ghost** [rlightner/public-BrewStatus#32]
This is a feature request more than an issue.

I was wondering if it was possible to make it so that the graphs showing Tilt data could be limited to only show data from between the times you set as the start and end of your brew?

Reason being, I cold crash my beer after fermentation completes, and I like to keep the tilt in there to monitor the temperature during that time. However the temp change produces incorrect gravity readings which throws out the overall brew info. If I could set the end of the brew date to when fermentation actually finished once that completed it would give the correct overall info.

**Comment by rlightner** [comment]
Yes. Totally agree with this. I usually just go and delete the data but it would be an easier way to do it. I'll try and get this out in the next release.

**Comment by ghost** [comment]
Great, thanks for this!

**Comment by elogicmedia** [comment]
I was wondering if a feature could be a starting cold condition date (or a stop gravity checkbox) that stops logging the gravity reading and continues to log temperature reading as an option to this? Above would also be fine, just a suggestion.

Thanks
